Evaluator.java

```diff
@@ -2,4 +2,16 @@ public class Evaluator {
     public static void main(String[] args) {
         System.out.println("Hello, Monkey Butt!");
     }
+
+    public class Equation {
+        private int operand_1, operand_2;
+        private char operator;
+
+        public Equation(int operand_1, int operand_2, char operator) {
+            this.operand_1 = operand_1;
+            this.operand_2 = operand_2;
+            this.operator = operator;
+        }
+
```

Comment:
Please remove this extra space here. Otherwise it looks good

Reply:
okayyyyy fine